op-program/host/config/config.go

```diff
@@ -229,3 +246,19 @@ func loadChainConfigFromGenesis(path string) (*params.ChainConfig, error) {
 	}
 	return genesis.Config, nil
 }
+
+func loadRollupConfig(rollupConfigPath string) (*rollup.Config, error) {
+	file, err := os.Open(rollupConfigPath)
+	if err != nil {
+		return nil, fmt.Errorf("failed to read rollup config: %w", err)
+	}
+	defer file.Close()
+
+	var rollupConfig rollup.Config
+	dec := json.NewDecoder(file)
+	dec.DisallowUnknownFields()
+	if err := dec.Decode(&rollupConfig); err != nil {
+		return nil, fmt.Errorf("failed to decode rollup config: %w", err)
+	}
+	return &rollupConfig, nil
+}
```

Comment:
It would be good to avoid duplicating this parsing logic.  We've got it here and in `chaincfg.rollupConfigByChainID` and in `opnode.NewRollupConfig`.  It would probably be worth having a `ParseRollupConfig(in io.Reader)` function alongside the `rollup.Config` struct so all these places can use it.

Also noticed that the `chaincfg.rollupConfigByChainID` version is missing a `defer file.Close()`.